src/meta/model/migration/Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = { workspace = true }
 repository = { workspace = true }
 
 [dependencies]
-async-std = { version = "1", features = ["attributes", "tokio1"] }
```

Comment:
Seems we only need an async main here. Why not just use tokio directly rather than introducing async-std/smol?

Reply:
+1

Reply:
Yes, you're right. async-std should have been the original seaorm migration auto-generated or copied-pasted from example. Here we just need to replace it with tokio. Let me change it.